toolkits/asana/arcade_asana/tools/users.py

```diff
@@ -0,0 +1,54 @@
+from typing import Annotated, Any
+
+from arcade.sdk import ToolContext, tool
+from arcade.sdk.auth import OAuth2
+
+from arcade_asana.constants import USER_OPT_FIELDS
+from arcade_asana.models import AsanaClient
+from arcade_asana.utils import clean_request_params
+
+
+@tool(requires_auth=OAuth2(id="arcade-asana", scopes=["default"]))
+async def list_users(
```

Comment:
When I call this tool with no inputs provided:

```
OutputError(message='Need to specify a workspace to paginate!', additional_prompt_content=None, can_retry=None, developer_message='Need to specify a workspace to paginate! | For more information on API status codes and how to handle them, read the docs on errors: https://developers.asana.com/docs/errors (HTTP status code: 400)', retry_after_ms=None)
```

Reply:
forgot to use `get_unique_workspace_id_or_raise_error` here if the workspace_id is not provided